media-libs/oyranos/oyranos-9999.ebuild

```diff
@@ -1,35 +1,35 @@
 # Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
-# $Header: /var/cvsroot/gentoo-x86/media-libs/oyranos/oyranos-9999.ebuild,v 1.7 2015/05/27 20:31:04 tamiko Exp $
+# $Header: $
 
 EAPI=5
 
 inherit eutils flag-o-matic cmake-utils cmake-multilib
 
-DESCRIPTION="colour management system allowing to share various settings across applications and services"
+DESCRIPTION="Colour management system allowing to share various settings across applications and services"
 HOMEPAGE="http://www.oyranos.org/"
 if [[ ${PV} == "9999" ]] ; then
 	EGIT_REPO_URI="https://github.com/${PN}-cms/${PN}.git"
 	inherit git-r3
 	KEYWORDS=""
 else
-	SRC_URI="mirror://sourceforge/oyranos/Oyranos/Oyranos%200.9/${P}.tar.bz2"
+	SRC_URI="https://github.com/oyranos-cms/oyranos/archive/${PV}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~x86"
 fi
 
 LICENSE="BSD"
 SLOT="0"
-IUSE="X cairo cups doc exif fltk qt4 raw test"
+IUSE="X cairo cups doc exif fltk jpeg qt4 qt5 raw test tiff"
 
-RDEPEND="
+#OY_LINGUAS="cs;de;eo;eu;fr;ru" #TODO
+
+CDEPEND="
```

Comment:
Please don't call it `CDEPEND`. It's safer to reserve short *DEPEND for future variables and use something like `COMMON_DEPEND`.